### PR TITLE
Raptor Changes:

### PR DESCRIPTION
**Pull request by EidolonForgery**
+ Gunner can now use Cargo interaction
+ Cargo space now set to 6
+ Inventory now has backup items
  - Rifles in inventory do not have attachments by default

Script Component Change:
+ Transport Items macros added

### CODE REVIEW

File: addons/vehicles/script_component.hpp
Comment (WitchFreya) [inline addons/vehicles/script_component.hpp:41]: These already exist in addons\main\script_macros but you're welcome to rename those and refactor them elsewhere if you want to use this syntax instead

File: addons/vehicles/CfgVehicles/Raptor.inc
Comment (WitchFreya) [inline addons/vehicles/CfgVehicles/Raptor.inc:11]: Looks like an errant comment slipped in.

**Review by WitchFreya — CHANGES_REQUESTED** [review]
mostly good, small tweaks

File: addons/vehicles/script_component.hpp
Comment (EidolonForgery) [reply addons/vehicles/script_component.hpp:41]: Oh, I'll just use the base ones then. I didn't think to check that they were already defined, I should look at that file more often.

**Review by WitchFreya — APPROVED** [review]
lgtm, once the build annotations are done you can merge it

**Comment by WitchFreya** [comment]
in the squashed commit make the message something like
`feat(vehicles): Update Raptor cargo`

**Pull request merged**
